patches.json

```diff
@@ -70,7 +70,9 @@
     },
     "Add Zookeeper and flock locks": {
       "2.2.5 - 2.2.8": "MAGECLOUD-3054__add_zookeeper_and_flock_locks__2.2.5.patch",
-      "2.3.0 - 2.3.1": "MAGECLOUD-3054__add_zookeeper_and_flock_locks__2.3.0.patch"
+      "2.3.0": "MAGECLOUD-3054__add_zookeeper_and_flock_locks__2.3.0.patch",
+      "2.3.1": "MAGECLOUD-3054__add_zookeeper_and_flock_locks__2.3.1.patch",
+      ">=2.3.2 <2.3.5": "MCLOUD_6139__improvement_flock_locks__2.3.2.patch"
```

Comment:
If this is now 2.3.0 - 2.3.5 - This should be ">=2.3.2 <2.3.6" ?

Reply:
This patch `MCLOUD_6139__improvement_flock_locks__2.3.2.patch` contains changes for Magento `2.3.2` - `2.3.4-px` where `x` some number.  So, we cannot use a constraint like this `2.3.2 - 2.3.4` and we use this one `>=2.3.2 <2.3.5`

Reply:
Okay that's fine - just wanted to check that was the reason!